cmd/orchestrate/orchestrate.go

```diff
@@ -302,3 +356,42 @@ func mappingMessage(body []byte, conf *config.Config) ([]byte, interface{}) {
 
 	return publish, new(mapping)
 }
+
+func releaseMessage(body []byte, conf *config.Config) ([]byte, interface{}) {
+	var message finalize
+	if err := json.Unmarshal(body, &message); err != nil {
+		return nil, nil
+	}
+	datasetID := uuid.NewSHA1(
+		uuid.NewSHA1(uuid.NameSpaceDNS, []byte(conf.Orchestrator.ProjectFQDN)),
+		body).URN()
+
+	msg := mapping{
+		Type:      "release",
+		DatasetID: datasetID,
+	}
+
+	publish, _ := json.Marshal(&msg)
+
+	return publish, new(mapping)
+}
+
+func validateMsg(delivered *amqp091.Delivery, mq *broker.AMQPBroker, routingKey string, durable bool, routingSchema string, publishMsg []byte, publishType interface{}) error {
+	err := mq.ValidateJSON(delivered, routingSchema, publishMsg, publishType)
+	if err != nil {
+		return nil
```

Comment:
Is this correct ("invert error state")? If so, could we have a comment about why?

Reply:
that looks like a miss on my part, good catch fixing ...